host/xtest/xtest_20000.c

```diff
@@ -335,8 +344,11 @@ static int obj_corrupt(TEEC_UUID *p_uuid, void *file_id,
 		}
 
 		res = stat(name, &st);
-		if (res < 0)
+		if (res < 0) {
+			perror("Failed to get file information");
```

Comment:
`fprintf(stderr, "stat(\"%s\"): %s", strerror(errno));` is preferable
Please apply the corresponding pattern for all the other cases with perror() below.